yarn.lock

```diff
@@ -7735,7 +7735,7 @@ fb-watchman@^2.0.0:
   dependencies:
     bser "^2.0.0"
 
-fbemitter@^2.1.1:
+fbemitter@2.1.1, fbemitter@^2.1.1:
```

Comment:
Thanks for removing the `fbemitter` dependency. I guess this file shouldn't change then?